src/config/app.config.test.ts

```diff
@@ -79,57 +77,24 @@ describe('appConfig', () => {
     });
   });
 
-  describe('parsing SINGLE_LINE_LOG_FORMAT', () => {
```

Comment:
We probably want this back in?

Reply:
I moved environment variable SINGLE_LINE_LOG_FORMAT test to helper withEnvironmentVariableParsingUnitTests